recipes/pymvpa2/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set name = "pymvpa2" %}
+{% set version = "2.6.1" %}
+{% set sha256 = "206fa302fc6292831c50d6ffe9b1a95fc048380b0701c140eb7300dcd017052a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py>3]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  # TODO: entry_points -- well, we have scripts
```

Comment:
So no `entry_points` yet then?

Reply:
yeah, forgot that we have no entry_points there and only scripts ATM, I guess this comment can (will) be safely removed ;)